src/app/components/study-accordion/study-accordion.directive.js

```diff
@@ -64,4 +64,11 @@ angular.module('transmartBaseUi')
         }
       };
 
+      $scope.clearMetadata = function () {
+        $scope.metadataObj = {};
+        var query = document.getElementsByClassName("popover ng-isolate-scope right fade in");
```

Comment:
just tried with the single popover selection, it works. I will push it to the PR.